### ISSUE TITLE
Can't include "*" in workload alias

### ISSUE THREAD

**Issue by iamhopaul123** [aws/copilot-cli#3736]
Right now if the workload alias contains "*", Copilot throws a weird error as below
```
✘ generate workload frontend template against environment prod: generate stack template: apply task definition overrides: unmarshal YAML template: yaml: line 488: did not find expected alphabetic or numeric character
```

**Comment by efekarakus** [comment]
Can you elaborate? What is a sample input that you would have expected to work but get this error instead

**Comment by iamhopaul123** [comment]
Sample input would be
```yaml
http:
  alias: *.example.com
```

**Comment by efekarakus** [comment]
This fix is now out in v1.20 🚀 
Release notes: https://github.com/aws/copilot-cli/releases/tag/v1.20.0 !
Blog post: https://aws.github.io/copilot-cli/blogs/release-v120/
